src/net.cpp

```diff
@@ -1602,7 +1601,7 @@ void CConnman::ThreadDNSAddressSeed()
         LOCK(cs_vNodes);
         int nRelevant = 0;
         for (auto pnode : vNodes) {
-            nRelevant += pnode->fSuccessfullyConnected && ((pnode->nServices & nRelevantServices) == nRelevantServices);
```

Comment:
I think services are just acting as a proxy here. Since we enforce service checking for (non-manual) outgoing connections already, and should probably consider incoming nodes as insignificant for the sake of this check, don't we really just want to tally ```fSuccessfullyConnected && !fInbound && !m_manual_connection``` ?

Reply:
Agreed, though I went ahead and added OneShot and Feeler into the mix.